protos/batch.proto

```diff
@@ -17,6 +17,7 @@ syntax = "proto3";
 
 option java_multiple_files = true;
 option java_package = "sawtooth.sdk.protobuf";
+option go_package = "sawtooth_sdk/protobuf";
```

Comment:
Or you can see the ordering here: https://github.com/aludvik/sawtooth-core/commits/install_go